bpf/lib/common.h

```diff
@@ -221,9 +221,8 @@ struct endpoint_key {
 	union {
 		struct {
 			__u32		ip4;
-			__u32		pad1;
-			__u32		pad2;
-			__u32		pad3;
+			__u32		vni;
+			mac_t		dmac;
```

Comment:
Can you add some details int he commit message on where these new fields are populated. Its not immediately obvious from the patch diff.

Reply:
I used bpftool wrapped in a bash script https://github.com/vincentmli/docs/blob/master/cilium-bigip/tunnel.sh to  populate the map, ideally 'cilium bpf tunnel' command  should be able to do that, but current cilium bpf tunnel command seem unable to update the map, my golang skill is limited now to improve cilium bpf tunnel command

Reply:
yes, I will add the bpftool update comment in commit message